### DISCUSSION TITLE
Incorrect radiant location for Southern delta-Aquariids?

### DISCUSSION THREAD

**Discussion by ArnoldBrody** [Stellarium/stellarium#1773] [Q&A] [answered]
In Stellarium 0.21.1, the radiant for the Southern δ-Aquariids is displayed inside the borders of Capricornus at RA 21h 40m DEC -17.43°.  On-line resources give an RA around 22h 36m, 13.5° to the east, in Aquarius.

Is there a way to correct this?

Thank you.

**Comment by gzotti** [comment]
The meteor shower plugin is also using "online resources". Which are yours?

**Comment by ArnoldBrody** [comment, marked as answer]
My mistake. I discovered that radiants move every night. There is no problem.

Thanks,
